web/unit_tests/cases/events.js

```diff
@@ -48,9 +47,11 @@ describe('Event Management', function() {
     if(focusEvent) {
```

Comment:
Can `focusEvent` ever be falsy here? I'd not think so -- in which case the `if()` is unnecessary and unhelpful